tests/e2e/tests/test_ack.py

```diff
@@ -0,0 +1,88 @@
+import subprocess
+import pytest
+
+from e2e.utils.constants import DEFAULT_USER_NAMESPACE
+from e2e.utils.config import metadata, configure_resource_fixture, configure_env_file
+
+from e2e.conftest import region
+
+from e2e.fixtures.cluster import cluster
+from e2e.fixtures.kustomize import kustomize, clone_upstream #, configure_manifests
+from e2e.fixtures.profile_dependencies import (
+    configure_manifests,
+    profile_controller_policy,
+    profile_controller_service_account,
+    profile_trust_policy,
+    profile_role,
+    associate_oidc,
+    kustomize_path,
+)
+from e2e.fixtures.clients import (
+    account_id,
+    kfp_client,
+    port_forward,
+    session_cookie,
+    host,
+    login,
+    password,
+)
+from e2e.fixtures.notebook_dependencies import notebook_server
+
+TO_ROOT_PATH = "../../"
+CUSTOM_RESOURCE_TEMPLATES_FOLDER = "./resources/custom-resource-templates"
+
+NOTEBOOK_IMAGES = [
+    "public.ecr.aws/c9e4w0g3/notebook-servers/jupyter-tensorflow:2.6.3-cpu-py38-ubuntu20.04-v1.8",
+]
+
+testdata = [
+    ("ack", NOTEBOOK_IMAGES[0], "verify_ack_integration.ipynb", "No resources found in kubeflow-user-example-com namespace"),
+]
+
+# GENERIC_KUSTOMIZE_MANIFEST_PATH = "../../deployments/vanilla"
+
+# @pytest.fixture(scope="class")
+# def kustomize_path():
+#     return GENERIC_KUSTOMIZE_MANIFEST_PATH
+
+class TestACK:
```

Comment:
need to add the the test as part of to test_sanity_portforward, it needs to run in canaries

Reply:
oh is that the only way to run it in canaries ? And do we need to add both tests - ack and kfp or just one?